src/widgets/configure/configure.js

```diff
@@ -0,0 +1,50 @@
+import isPlainObject from 'lodash/isPlainObject';
+const usage = `Usage:
+search.addWidget(
+  instantsearch.widgets.configure({
+    // any searchParameter
+  })
+);
+Full documentation available at https://community.algolia.com/instantsearch.js/widgets/configure.html
+`;
+
+/**
+ * The **Configure** widget provides the logic to build a custom widget that
```

Comment:
It does not provide logic since it's a widget and not a connector anymore. What about:

> The **configure** widget is a headless widget that let you configure the settings of your search using the parameters described by [the general Algolia documentation](https://www.algolia.com/doc/api-reference/search-api-parameters/)